tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -110,9 +110,14 @@
 #endif
 
 static const struct dac_channel_cfg dac_ch_cfg = {
-	.channel_id  = DAC_CHANNEL_ID,
-	.resolution  = DAC_RESOLUTION,
-	.buffered = true
+	.channel_id = DAC_CHANNEL_ID,
+	.resolution = DAC_RESOLUTION,
+#if defined(CONFIG_BOARD_EK_RA4L1) || defined(CONFIG_BOARD_EK_RA4M1) ||                            \
```

Comment:
I think this situation may occur on other boards as well.
It seems better to add config to Kconfig.